docs/get_started.rst

```diff
@@ -1,17 +1,341 @@
+#. At the top of your GitHub Repository's Quick Setup page, click the clipboard button to copy the remote repository URL.
+#. In your terminal, `add the URL for the remote repository <https://docs.github.com/en/free-pro-team@latest/articles/adding-a-remote>`_ to where your local repository will be pushed::
+
+    $ git remote add origin <remote_repository_url>
+
+#. Verify the remote was added successfullly with::
+
+    $ git remote -v
+
+#. `Push the changes <https://docs.github.com/en/free-pro-team@latest/github/using-git/pushing-commits-to-a-remote-repository>`_ in your local repository to GitHub::
+
+    $ git push -u origin master
+
+
+
+Convert an Existing Salesforce Project
+--------------------------------------
+Converting an existing Salesforce project to use CumulusCI may follow a number of different paths, depending on whether you're practicing the Org Development Model or the Package Development Model, whether or not you're already developing in scratch orgs, and the complexity of your project's dependencies on the org environment.
+If you're coming from developing on scratch orgs, then you likely only need to do `project setup`_ and `org shape setup`.
+If you're working out of persistent orgs, then you will likely want to go through *all* of the following sections.
+Your experience may vary.
+You're welcome to discuss project conversion in the `CumulusCI Trailblazer group <https://trailblazers.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009M9Z>`_.
+
+
+Project Setup
+^^^^^^^^^^^^^
+#. Create a directory for your project to live in, and navigate to it::
+
+    $ mkdir mySalesforceProject; cd mySalesforceProject
+
+#. Initialize the directory as a git repository::
+
+    $ git init
+    Initialized empty Git repository in /Users/MrCCI/repos/mySalesforceProject/.git/
+
+#. Initialize the repository as a CumulusCI project. See `project initialization`_.
+
+
+
+Metadata Capture
+^^^^^^^^^^^^^^^^
+We're assuming that your project currently lives in a persistent org.
+We recommend a retrieve of MetaData via the MetaData API (via ``sfdx``), followed by converting the source format from "metadata" to "``sfdx``".
+
+#. `Create a package <https://help.salesforce.com/articleView?id=creating_packages.htm&language=en_us&r=https:%2F%2Fwww.google.com%2F&type=5>`_ in the target org.
+    * Ensure that the package namespace matches the namespace you entered when running ``cci project init``.
+#. Run the `retrieve command <https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_mdapi.htm#cli_reference_retrieve>`_ to extract your package metadata::
+
+    $ sfdx force:mdapi:retrieve -p package_name -r /path/to/project/ 
+
+#. Navigate to your projects root directory (i.e. where the ``src/`` folder lives), and you can now convert your metadata to source (``sfdx``) format with ``cci``::
```

Comment:
```suggestion
#. Navigate to your project's root directory (i.e. where the ``src/`` folder lives), and you can now convert your metadata to source (``sfdx``) format with ``cci``::
```

Reply:
Use `force:source:retrieve` instead, with the -n option, and then there's no need for a separate conversion step.